test/013_oracles_composite.ts

```diff
@@ -101,4 +111,56 @@ describe('Oracles - Composite', function () {
     expect((await compositeMultiOracle.peek(bytes6ToBytes32(DAI), bytes6ToBytes32(USDC), WAD))[0]).to.equal(oneUSDC)
     expect((await compositeMultiOracle.peek(bytes6ToBytes32(USDC), bytes6ToBytes32(DAI), oneUSDC))[0]).to.equal(WAD)
   })
+
```

Comment:
I would move the testing of the Lido oracle to it's own testing file (`016_oracles_lido.ts`) and leave the composite test file to just do basic tests on the composite token.